.github/workflows/ci.yml

```diff
@@ -20,10 +20,27 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: shellcheck ./zig/download.sh
-      - run: ./zig/download.sh
-      - run: ./zig/zig fmt --check .
-      - run: ./zig/zig build check
-      - run: ./zig/zig build scripts -- ci --build-docs
+      - run: ./zig/download.ps1 && ./zig/zig build ci -- smoke
+
+  test:
+    strategy:
+      matrix:
+        include:
+          - { os: 'ubuntu-latest' }
+          - { os: 'ubuntu-latest-arm64' }
+          - { os: 'windows-latest' }
```

Comment:
Out-of-scope for PR, but perhaps we should add Windows ARM64 too.

Reply:
We don't support building for arm windows, and I'd rather add that only when someone asks. I am confident that we _could_ support whatever, but I want to minimize the time to administrate CI heterogeneity (my mental model is that any CI config has an independent probability of "flakes" for boring reasons).